stor/s3.py

```diff
@@ -601,7 +594,7 @@ def download(self, dest, condition=None, use_manifest=False, **kwargs):
                     except StopIteration:
                         break
                 pool.close()
-            except:
+            except BaseException:
```

Comment:
is there a difference between this and previous version?

https://stackoverflow.com/a/7161517

Reply:
No, but linter complains otherwise. Also I appreciate that it indicates to reader we want to catch KeyboardInterrupt and friends